backend/btrixcloud/crawls.py

```diff
@@ -210,51 +234,97 @@ async def list_crawls(
         if running_only:
             query["state"] = {"$in": ["running", "starting", "stopping"]}
 
+        # Override running_only if state list is explicitly passed
+        if state:
+            validated_states = [value for value in state if value in CRAWL_STATES]
```

Comment:
By default, we list only the running states, and this allows for override with a custom list of states, right?

Reply:
By default we allow all states if `running_only=False` and only running states if `running_only=True` (e.g. in admin view). If a list is passed to `state`, that list acts as an override, yes.